projects/demo/src/pages/phone/examples/4-lazy-metadata/simple.md

```diff
@@ -0,0 +1,19 @@
+```js
+import {Maskito, MASKITO_DEFAULT_OPTIONS} from '@maskito/core';
+import {maskitoPhoneOptionsGenerator} from '@maskito/phone';
+
+const element = document.querySelector('input,textarea');
+let maskedInput = new Maskito(element, MASKITO_DEFAULT_OPTIONS);
```

Comment:
nit: maybe add an empty line before it? Would probably read better. BTW, what's the benefit of initializing it with empty mask and then recreating mask, why not just leave it be until we get metadata?